src/tracker-component.js

```diff
@@ -246,6 +255,14 @@ const track = <T>(config : Config, trackingType : TrackingType, trackingData : T
     }
 };
 
+const clearTrackQueue = config => {
+    // eslint-disable-next-line array-callback-return
+    window.trackEventQueue = window.trackEventQueue.filter(eventData => {
```

Comment:
Thanks for the fix 👍 Can we add a test for this if we don't have one? I think the behavior would've been caught with that